src/openai-auth.ts

```diff
@@ -125,15 +127,23 @@ export async function getOpenAIAuth({
  * recognizes it and blocks access.
  */
 export async function getBrowser(launchOptions?: PuppeteerLaunchOptions) {
-  const macChromePath =
-    '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  let executablePath
+
+  if (/^win/i.test(osPlatform)) {
+    executablePath = ' C:\\ProgramFiles\\Google\Chrome\\Application\\chrome.exe'
+  } else if (/^darwin/i.test(osPlatform)) {
+    executablePath =
+      '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  } else if (/^linux/i.test(osPlatform)) {
```

Comment:
make this a general `else` without the `linux` test because there are other unix values aside from just `linux`: https://nodejs.org/api/os.html#osplatform